### PR TITLE
chore(server): mark old raft configs as deprecated

### PR DESCRIPTION
**Pull request by shirleyStorage**
## Purpose of the PR

- close https://github.com/apache/incubator-hugegraph/issues/2660

## Main Changes

Place the raft configurations at the bottom of the properties file and mark it as being deprecated.

## Verifying these changes

- [x] Trivial rework / code cleanup without any test coverage. (No Need)
- [ ] Already covered by existing tests, such as *(please modify tests here)*.
- [ ] Need tests and can be verified as follows:
    - xxx

## Does this PR potentially affect the following parts?

- [ ]  Dependencies ([add/update license](https://hugegraph.apache.org/docs/contribution-guidelines/contribute/#321-check-licenses) info & [regenerate_dependencies](../hugegraph-server/hugegraph-dist/scripts/dependency/regenerate_known_dependencies.sh)) 
- [ ]  Modify configurations
- [ ]  The public API
- [ ]  Other affects (typed here)
- [x]  Nope

## Documentation Status

- [ ]  `Doc - TODO` 
- [ ]  `Doc - Done` 
- [x]  `Doc - No Need`

### CODE REVIEW

**Comment by shirleyStorage** [comment]
@imbajin PTAL~
This PR is related to "Mark raft-mode configs as legacy (put them on the bottom - properties)" of  https://github.com/apache/incubator-hugegraph/issues/2575

File: hugegraph-server/hugegraph-dist/src/assembly/static/conf/graphs/hstore.properties.template [hunk 1]
```diff
@@ -107,3 +85,25 @@ cassandra.password=
...
+raft.snapshot_compress_threads=4
+raft.snapshot_decompress_threads=4
+raft.read_strategy=ReadOnlyLeaseBased
+raft.queue_size=16384
+raft.queue_publish_timeout=60
+raft.apply_batch=1
+raft.rpc_threads=80
+raft.rpc_connect_timeout=5000
+raft.rpc_timeout=60
+raft.install_snapshot_rpc_timeout=36000
```
Comment (imbajin) [inline hugegraph-server/hugegraph-dist/src/assembly/static/conf/graphs/hstore.properties.template:109]: comment them as default `# raft*` (Also add "Warning")

File: hugegraph-server/hugegraph-dist/src/assembly/static/conf/graphs/hstore.properties.template [hunk 2]
```diff
@@ -107,3 +85,25 @@ cassandra.password=
 #palo.poll_interval=10
 #palo.temp_dir=./palo-data
 #palo.file_limit_size=32
+
+# These raft configurations will be deprecated, please use the official version instead
```
Comment (imbajin) [inline hugegraph-server/hugegraph-dist/src/assembly/static/conf/graphs/hstore.properties.template:89]:
```suggestion
# WARNING: These raft configurations are deprecated, please use the latest version instead
```

File: hugegraph-server/hugegraph-dist/src/assembly/static/conf/graphs/hugegraph.properties [hunk 3]
```diff
@@ -107,3 +85,25 @@ cassandra.password=
...
+raft.snapshot_compress_threads=4
+raft.snapshot_decompress_threads=4
+raft.read_strategy=ReadOnlyLeaseBased
+raft.queue_size=16384
+raft.queue_publish_timeout=60
+raft.apply_batch=1
+raft.rpc_threads=80
+raft.rpc_connect_timeout=5000
+raft.rpc_timeout=60
+raft.install_snapshot_rpc_timeout=36000
```
Comment (imbajin) [inline hugegraph-server/hugegraph-dist/src/assembly/static/conf/graphs/hugegraph.properties:109]: same (comment them)

**Review by VGalaxies — APPROVED** [review]

**Review by imbajin — APPROVED** [review]

**Pull request merged**